<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>715</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>575</width>
    <height>715</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>575</width>
    <height>715</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fragebogen</string>
  </property>
  <property name="locale">
   <locale language="German" country="Germany"/>
  </property>
  <widget class="QLabel" name="questionnaire_intro">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Vielen Dank für die Teilnahme!&lt;/p&gt;&lt;p&gt;Bitte nimm dir noch kurz Zeit, um den folgenden Fragebogen auszufüllen:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="finish_study_btn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>670</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(91, 208, 1);</string>
   </property>
   <property name="text">
    <string>Senden</string>
   </property>
  </widget>
  <widget class="QComboBox" name="gender_selection">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>87</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Männlich</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Weiblich</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Divers</string>
    </property>
   </item>
  </widget>
  <widget class="QSlider" name="keyboard_affinitiy_slider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>511</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>7</number>
   </property>
   <property name="pageStep">
    <number>10</number>
   </property>
   <property name="sliderPosition">
    <number>4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="age_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Dein Alter:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="age_selection">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>52</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimum">
    <number>18</number>
   </property>
   <property name="maximum">
    <number>120</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="gender_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Geschlecht:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="keyboard_affinitiy_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>511</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Bitte gib an, wie häufig du im Alltag eine Tastatur verwendest:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="keyboard_affinity_low">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>510</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>sehr selten</string>
   </property>
  </widget>
  <widget class="QLabel" name="keyboard_affinity_high">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>510</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>sehr häufig</string>
   </property>
  </widget>
  <widget class="QLabel" name="occupation_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Beruf :&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="occupation_input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>521</width>
     <height>32</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="color_deficiency_selection">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>87</width>
     <height>32</height>
    </rect>
   </property>
   <property name="currentText">
    <string>Nein</string>
   </property>
   <item>
    <property name="text">
     <string>Ja</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nein</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="color_deficiency_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>461</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Hast du eine Farbenfehlsichtigkeit (z.B. eine Rot-Grün-Sehschwäche) ?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="color_deficiency_yes_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>620</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Wenn ja, welche:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="color_deficiency_input">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>610</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="hand_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>341</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Mit welcher Hand hast du die Aufgaben bearbeitet ?&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="hand_selection">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Rechte Hand (starke Hand)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rechte Hand (schwache Hand)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Linke Hand (starke Hand)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Linke Hand (schwache Hand)</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="keyboard_type_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>541</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Was für eine Tastatur hast du für die Studie verwendet (z.B. extern, Laptop-Tastatur, ...) ?&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyboard_type_input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>521</width>
     <height>32</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
